app/cafe.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
```

Comment:
remove redundant import

Reply:
I don't understand why annotations don't work without import at the beginning of writing the code, but after your comment they work?